test/Organizer/OrganizerTest.php

```diff
@@ -285,6 +286,40 @@ function (Organizer $organizer) use ($initialAddress, $updatedAddress, $language
             );
     }
 
+    /**
+     * @test
+     */
+    public function it_can_set_an_initial_address_and_remove_it_later()
+    {
+        $initialAddress = new Address(
+            new Street('Wetstraat 1'),
+            new PostalCode('1000'),
+            new Locality('Brussel'),
+            Country::fromNative('BE')
+        );
+
+        $language = $this->organizerCreatedWithUniqueWebsite->getMainLanguage();
+
+        $this->scenario
+            ->given([$this->organizerCreatedWithUniqueWebsite])
+            ->when(
+                function (Organizer $organizer) use ($initialAddress, $language) {
+                    $organizer->updateAddress($initialAddress, $language);
```

Comment:
Technically this works, but semantically maybe it should be part of the `given` instead.
(So by adding an `AddressUpdated` event to the array in `given`.)